trade-finance-manager-ui/server/controllers/helpers/deal-cancellation-enabled.helper.test.ts

```diff
@@ -26,3 +26,23 @@ describe('dealCancellationEnabled', () => {
     });
   });
 });
+
+describe('canSubmissionTypeBeCancelled', () => {
+  it('returns true when deal submission type is AIN', () => {
```

Comment:
One minor not a blocker, might be effective to use `it.each(...`

Reply:
This is now implemented, thanks!👍